ext/drx/drx.c

```diff
@@ -66,6 +66,10 @@
 #define XMM_REG_SIZE 16
 #define MAX(x, y) ((x) >= (y) ? (x) : (y))
 
+#ifdef X86
+#    define PLATFORM_SUPPORTS_SCATTER_GATHER
```

Comment:
There should be comments and an i# about adding SIMD ARM.

Reply:
Done.